APLSource/Main-1/ProcessCreateArg-2081.aplf

```diff
@@ -0,0 +1,9 @@
+ ProcessCreateArg←{
+    ⍝ ⍵ ←→ Folder, Space (one character vector)
```

Comment:
Hmm, do we want to allow ]dado.createproject to take the same arguments as ]acre.createproject (-variables/-casecode/-keephistory)? If so, I'll have to rework how this is done. Remember, folder paths can have spaces in them which is why i search from the end of the array. I guess we simply could change this to look for the "-"'s as well